clang/test/3C/allarrays.c

```diff
@@ -1,5 +1,5 @@
 // RUN: 3c -alltypes %s | FileCheck -match-full-lines --check-prefixes="CHECK" %s
-// RUN: 3c -alltypes %s | %clang -c -f3c-tool -fcheckedc-extension -x c -o /dev/null -
+// RUN: 3c -alltypes %s | %clang -c -f3c-tool -fcheckedc-extension -x c -o %t1.unused -
```

Comment:
Why do you need the `.unused` bit at the end? I presume you've confirmed that this file is deleted after all the tests are run?

Reply:
I don't need it, but I wanted an indication of what this particular temp file is used for.